plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointEntryIterator.java

```diff
@@ -763,6 +763,11 @@ private boolean tryAdvancePage()
         return true;
     }
 
+    public void close()
```

Comment:
This is added in "Ensure the checkpoint transaction log streams are closed after usage"
but unused in the commit

move it to the later one